src/task-2-2.js

```diff
@@ -0,0 +1,78 @@
+var { join, extname, basename } = require('path');
+var { createWriteStream } = require('fs');
+var { readdir, mkdir } = require('fs');
+
+var csv = require('csvtojson');
+
+// check csv folder on .csv files and work with them if they exists
+function csvManager() {
```

Comment:
naming conventions
Functions
Use descriptive names, usually verbs in the imperative form.

Reply:
this name is pretty descriptive, don't know what you ain't like there
this function manage files with `.csv` extension and so I named it `csvManager`, what's wrong with that? :)
when you see this name you immediately understand that this function do some work with csv